src/app/lib/views/movie_detail.js

```diff
@@ -1,220 +1,19 @@
 (function (App) {
     'use strict';
 
-    App.View.MovieDetail = Backbone.Marionette.LayoutView.extend({
-        template: '#movie-detail-tpl',
-        className: 'movie-detail',
-
-        ui: {
-            bookmarkIcon: '.favourites-toggle',
-            watchedIcon: '.watched-toggle',
-            backdrop: '.backdrop',
-            poster: '.mcover-image'
-        },
-
-        events: {
-            'click .go-back': 'closeDetails',
-            'click .movie-imdb-link': 'openIMDb',
-            'mousedown .magnet-link': 'openMagnet',
-            'click .rating-container': 'switchRating'
-        },
-
+    App.View.MovieDetail = App.View.GenericDetail.extend({
         regions: {
-            PlayControl: '#play-control'
+            ActionBar:  '#action-bar',
+            DetailMeat: '#detail-meat'
```

Comment:
yeah, but then it's called 'generic-detail', and we have 'detail', it's weird isn't it ?